patches/0036-Disable-retry-for-KMS-access-denied-in-lambda.patch

```diff
@@ -80,12 +81,12 @@ index 9437623666..741067f98f 100644
  	return &servicePackage{}
  }
 diff --git a/names/data/names_data.csv b/names/data/names_data.csv
-index c0ee031d37..a573761b55 100644
+index 3d567e120f..fae484a898 100644
 --- a/names/data/names_data.csv
 +++ b/names/data/names_data.csv
 @@ -218,7 +218,7 @@ kinesis-video-media,kinesisvideomedia,kinesisvideomedia,kinesisvideomedia,,kines
  kinesis-video-signaling,kinesisvideosignaling,kinesisvideosignalingchannels,kinesisvideosignaling,,kinesisvideosignaling,,kinesisvideosignalingchannels,KinesisVideoSignaling,KinesisVideoSignalingChannels,,1,,,aws_kinesisvideosignaling_,,kinesisvideosignaling_,Kinesis Video Signaling,Amazon,,x,,,,,Kinesis Video Signaling,,,
- kms,kms,kms,kms,,kms,,,KMS,KMS,,1,,,aws_kms_,,kms_,KMS (Key Management),AWS,,,,,,,KMS,ListKeys,,
+ kms,kms,kms,kms,,kms,,,KMS,KMS,,,2,,aws_kms_,,kms_,KMS (Key Management),AWS,,,,,,,KMS,ListKeys,,
```

Comment:
The CSV diff is a bit hard to read, did this change OK? This CSV drives some code-generator but we don't fully check that changes to the CSV keep the generated code consistent. I recently had to make some quick edits in there.

Reply:
This was changed upstream here: https://github.com/hashicorp/terraform-provider-aws/commit/8a2d87d8c680230e7c9a01c8d3e57423883f5726

How would I confirm whether this had any negative effects?

Reply:
I did some digging and the data in the CSV follows the format documented here: https://github.com/hashicorp/terraform-provider-aws/blob/main/names/README.md?plain=1#L34

What changed here was index 12, which indicates whether AWS SDKv2 is used or not. If SDK v2 is used, the value needs to be set to `2`. And that value was set to 2 now because they switched KMS over to use SDKv2: https://github.com/hashicorp/terraform-provider-aws/blame/main/internal/service/kms/service_package_gen.go#L132